icubam/messaging/message_formatter.py

```diff
@@ -0,0 +1,35 @@
+import os.path
+import tornado.locale
+import tornado.template
+
+
+class MessageFormatter:
+  """This class aims at formatting the messages sent to the user.
+
+  Formatting means turning the data in the message into a localized text or
+  html.
+  """
+  ROOT_PATH = os.path.abspath(__file__)
+  for _ in range(3):
+    ROOT_PATH = os.path.dirname(ROOT_PATH)
```

Comment:
This is also,
```py
from pathlib import Path

ROOT_PATH = Path(__file__).resolve().parents[2]
```

https://treyhunner.com/2018/12/why-you-should-be-using-pathlib/

Reply:
nice !